docs/admin/high-availability.md

```diff
@@ -6,19 +6,27 @@ endpoint. [GCP](https://cloud.google.com/sql/docs/postgres/high-availability), [
 and other cloud vendors offer fully-managed HA Postgres services that pair
 nicely with Coder.
 
-For Coder to operate correctly, every node must be within 10ms of each other
-and Postgres. We make a best-effort attempt to warn the user when inter-Coder
-latency is too high, but if requests start dropping, this is one metric to investigate.
+For Coder to operate correctly, Coderd instances should have low-latency connections
+to each other so that they can effectively relay traffic between users and workspaces no
+matter which Coderd instance users or workspaces connect to. We make a best-effort attempt
+to warn the user when inter-Coderd latency is too high, but if requests start dropping, this
+is one metric to investigate.
```

Comment:
suggestion: note specific metrics, link to https://coder.com/docs/v2/latest/admin/prometheus

Reply:
I don't think we expose this over Prometheus... @kylecarbs ?